pydarn/plotting/superdarn_cpid.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2019 SuperDARN
+# Author: Marina Schmidt
+
+"""
+This module contains SuperDARN radar modes classified by cpid number
+"""
+
+
+class SuperDARNCpids():
+
```

Comment:
This is fantastic.  Could there also be a longer description of these modes?  If it exists in documentation somewhere, adding that location to the docstring would be sufficient.

Reply:
Found out at SuperDARN Workshop the Scheduling Working Group is working on something around CP ID names. I also have a summer student creating a database to collect all CP ID's and their possible names from the `combf` field. So hopefully we can make some documentation around this, so we know what each mode means :) 

However, I am not the best person to be documenting these modes. Maybe it is something we can request from the Data Analysis Working Group to help with and put it in the readthedocs for RST since it comes from the files the software produces or further discussion with the Scheduling group.

Reply:
Found some documentation:
http://superdarn.thayer.dartmouth.edu/info/modes.html

Reply:
Added that to the code and put a TODO for better mode documentation. 

Going to call it done for now but I will create an Issue somewhere to address this.